File: Python/1151.py
anterior = proximo = 0
atual = 1

X = int (input())

for i in range (1,X+1):
    if i == X:
        print(anterior)
    else:
        print(f"{anterior} ",end="")
        proximo = atual + anterior
        anterior = atual
        atual = proximo

File: Java/1151.java
import java.util.Scanner;

public class 1151 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int anterior = 0, atual = 1, proximo = 0, X;

        X = input.nextInt();

        for (int i = 1; i <= X; i++) {
            if (i == X) {
                System.out.println(anterior);
            } else {
                System.out.print(anterior + " ");
                proximo = atual + anterior;
                anterior = atual;
                atual = proximo;

            }
        }

    }
}
